index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: setup and mobile version skeleton</title>
+</head>
+<body>
+
+    
+         <header>
+            <label id="logo">My Logo</label><br><br>
+        <nav class="manu">
+            <img id="img2" src="https://icon-library.com/images/menu-icon-png-3-lines/menu-icon-png-3-lines-10.jpg" alt="#" class="menu">
+    </nav>
+        </header>
+    <section class="headline">
+    <div class="flex-container">
+        <h1>Hey There I'm Alex</h1>
+        <h2>I'am a Software Developer</h2>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+    </div>
+   </section>
+           
+<img id="img1" src="illustration-header-mobile.png" alt="#">
+<div class="bottom">
+    <div class="columns-wrapper">
+        <div class="columns">
+<img id="img3" src="Vector.png" alt="#">
+<img id="img4" src="Group.png" alt="#">
+<img id="img5" src="1Vector.png" alt="#">
+<img id="img6" src="2Vector.png" alt="#">
+</div>
```

Comment:
- [ ] Good job in finding the right icons for this project. It will be nicer if we could wrap all the icons in a `ul` tag and a `li` per each icon. 

Example:
```
<ul>
   <li> // icon goes here </li>
</ul>
```